common/src/main/scala/com/gu/monitoring/MemberDataAPIMetrics.scala

```diff
@@ -0,0 +1,13 @@
+package com.gu.monitoring
+
+import Metrics._
+class MembersDataAPIMetrics(val name: String) extends CloudWatch(Seq(
+  productName(name)
```

Comment:
I don't think the extra newlines make this easier to read than
`class MembersDataAPIMetrics(val name: String) extends CloudWatch(Seq(productName(name))) {`